pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/IngestionDelayTracker.java

```diff
@@ -88,6 +90,16 @@ private static class IngestionTimestamps {
     private final long _ingestionTimeMs;
     private final long _firstStreamIngestionTimeMs;
   }
+
+  private static class IngestionOffsets {
+    IngestionOffsets(StreamPartitionMsgOffset offset, StreamPartitionMsgOffset latestOffset) {
+      _offset = offset;
+      _latestOffset = latestOffset;
+    }
+    private final StreamPartitionMsgOffset _offset;
+    private final StreamPartitionMsgOffset _latestOffset;
```

Comment:
nit: can we write the attributes before the constructor? Doing it the other way around is very strange.

Reply:
yeah, i was just following existing style. changing